### PR TITLE
Changed _repr_ string format to f-strings

### LINKED ISSUE #639: Change __repr__ strings format to f-strings
## Description
As a developer,
I need to have code using f-strings,
so that I can promote coding style consistency.

Coding standards 👉 https://github.com/anitab-org/mentorship-backend/wiki/Coding-Standards

## Mocks

Some of the places to change are:
- https://github.com/anitab-org/mentorship-backend/blob/develop/app/database/models/user.py#L115
- https://github.com/anitab-org/mentorship-backend/blob/develop/app/database/models/tasks_list.py#L158
- https://github.com/anitab-org/mentorship-backend/blob/develop/app/database/models/mentorship_relation.py#L99 (uncomment and change for f-strings)

### Update [Required]
- [ ] Change every place that is not using f-strings, at least in the places already talked about.

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
30 min

### PR DESCRIPTION
**Pull request by vj-codes**
### Description

Changed _repr_ string format to f-strings

Fixes #639 

### Type of Change:

- Code
- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by vj-codes** [comment]
Heyy @isabelcosta here's my first PR to the backend repo!
Quick question , I modified only 3 files but why are the virtual env files are also seen in changed files?Did I miss any step here?

**Review by satya7289 — COMMENTED** [review]
Why do you add files start with myenv/* & python files in the commits?
I guess you should not add these in the commit.

**Comment by vj-codes** [comment]
> Why do you add files start with myenv/* & python files in the commits?
> I guess you should not add these in the commit.

I didn't add them on purpose while I committed only the 3 modified files these were also added , I dont know how right now.I tried again from a different branch same thing happens .Any idea how to solve? @satya7289

**Comment by satya7289** [comment]
> > Why do you add files start with myenv/* & python files in the commits?
> > I guess you should not add these in the commit.
> 
> I didn't add them on purpose while I committed only the 3 modified files these were also added , I dont know how right now.I tried again from a different branch same thing happens .Any idea how to solve? @satya7289

Use git add file name(not use git add .)
And then commit.

File: myenv/Scripts/activate [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```
Comment (isabelcosta) [inline myenv/Scripts/activate]: These files where added because you probably ran `git add .` - which adds everything that is new to git - and also because the default name that we have on `.gitignore` for virtual environments is `venv` and you named it `myenv`.
Could you delete/rename that `myenv` to `venv`. I am asking that because it does not seem that it would scale well, you putting this on the `.gitignore` file. What you can do next is instead of performing `git add .` perform `git add <path/to/file>` for each of the files you which to commit :)

Makes sense @vj-codes ?

File: python [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ * Serving Flask app "run" (lazy loading)
```
Comment (isabelcosta) [inline python]: can you please remove this file? It does not seem to be related to your issue :)

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@vj-codes once you rename the folder to `venv` and activate that virtual environment, you will be able to commit again to remove the `myenv` from this branch :)

The rest seems totally fine and ready to be merged 😉

File: myenv/Scripts/activate (hunk 1 above)
Comment (vj-codes) [reply myenv/Scripts/activate]: Thanks @isabelcosta .I didn't know about using git add and specific path and it'll make such a huge difference.Glad learnt it now! 
I have removed all those files now , please have a look again.

File: python (hunk 2 above)
Comment (vj-codes) [reply python]: done

File: myenv/Scripts/activate (hunk 1 above)
Comment (isabelcosta) [reply myenv/Scripts/activate]: I am super glad that you learned something here 🎉 🤗  @vj-codes

**Review by isabelcosta — APPROVED** [review]
This looks good! Great first contribution to this repository @vj-codes 🎉 
Thank you for this :D

**Review by satya7289 — APPROVED** [review]
Look Great.
@isabelcosta does here needed the unit test case to verify that model returns the correct name of the object?

**Comment by isabelcosta** [comment]
@satya7289 I don't think its really necessary. This code is not necessarily used. I usually used in the beginning this for debugging. In fact this text may be removed in the future, we can do this in another issue. For now I think this is ok.

**Comment by isabelcosta** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Added a print in the middle of the code to see the representation of a user_  
    _Screenshot/gif_:  
image
    _Expected Result_:  the string appears in the console, without errors in the application
    _Actual Result_: the string appeared in the console

3. Additional testcases covered: N/A
    
4. Additional Comments:

I added this print line for testing this little change:
image

5. Status of PR Changed to: Ready to Merge.

**Comment by isabelcosta** [comment]
@vj-codes congratulations on your first merge in this repository! 🎉 
Thank you again for your contribution 👏
